yogstation/code/datums/traits/negative.dm

```diff
@@ -0,0 +1,5 @@
+/datum/quirk/family_heirloom
+	name = "Family Heirloom"
+	desc = "You are the current owner of an heirloom, passed down for generations. You have to keep it safe!"
```

Comment:
Why change name and desc?

Reply:
its the exact same I just didnt bother removing it one moment